ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -2655,12 +2655,6 @@ public sealed class ArchiWebHandler : IDisposable {
 				return (false, false);
 			}
 
-			if (userPrivacy == null) {
```

Comment:
I don't think this removal is okay, I can see how `null` json (not `{}`) could be deserialized into `null` object, we should keep this check.

Reply:
In that case an exception will be thrown, because required properties will be missing.

Reply:
Exactly, and we don't want Steam to kill our ASF process due to unhandled exception, neither it's expected to happen, which is why null error was there previously.

Reply:
Basically, `json` variable in our case can just have a value of `"null"` (string with null inside), which to the best of my knowledge will not throw exception during deserializing, but will instead deserialize it to our class with a value of `null`. I'm not talking about required properties from our JSON class missing, but about the entire object being absent.

Reply:
Like I said, deserializer won't throw exception here - it'll properly deserialize `"null"` json into `Steam.UserPrivacy` object of `null`.

Reply:
Because JSON string of `null` is a valid object that can be deserialized to any nullable entity, and `Steam.UserPrivacy` is one.

Reply:
I was confused by Rider, which was too sure that it can't be null :/

Reply:
I mean, you can check it yourself. Maybe I *am* wrong after all, but I remember this to be the case.

Reply:
Yes, I have checked manually, it looks like a bug.

Reply:
It's not a bug, that's intended behaviour - you're deserializing json value with a `null` into an object, it's exactly the same as if you did `var deserialized = (Steam.UserPrivacy) ((object) null)`, it'd also work just the way you imagine.

Reply:
I meant a bug in a static analyzer - I'm using preview version, and in the latest release version it is shown correctly, that the code can be reached.

Reply:
Ah, likely then, consider reporting it if you have nothing better to do.

Reply:
(It's probably worth checking that `htmlDocument` too then)